tests/parser/SimulationConfigTest.cpp

```diff
@@ -232,6 +241,22 @@ BOOST_AUTO_TEST_CASE(SimulationConfig_VAPOIL_DISGAS) {
     BOOST_CHECK_EQUAL( true , simulationConfig_vd.hasVAPOIL());
 }
 
+BOOST_AUTO_TEST_CASE(SimulationConfig_VAPWAT) {
+    auto deck = createDeck(inputStr);
+    TableManager tm(deck);
+    EclipseGrid eg(10, 3, 4);
+    FieldPropsManager fp(deck, Phases{true, true, true}, eg, tm);
+    SimulationConfig simulationConfig(false, deck, fp);
+    BOOST_CHECK_EQUAL( false , simulationConfig.hasVAPWAT());
+
+    auto deck_vd = createDeck(inputStr_watvap);
+    TableManager tm_vd(deck_vd);
+    EclipseGrid eg_vd(10, 3, 4);
```

Comment:
Can just reuse the same `EclipseGrid` instance?

Reply:
Yes, thank you. Just deleted eg_vd.